portfolio/src/main/webapp/comments/CommentFacade.js

```diff
@@ -37,9 +37,12 @@ class CommentFacade {
      * @async
      * @param name the name of the author of the comment
      * @param description the comment body
+     * @param images array of the image files associated with the comment
```

Comment:
The js doc should be much more specific and a little more structured. 

https://google.github.io/styleguide/jsguide.html#jsdoc-tags

So for images should be 

!Array<string> array of the urls of each of the images associated with the comment

Similar for the other files